modules/cas/lib/Auth/Source/CAS.php

```diff
@@ -207,6 +207,8 @@ public function finalStep(&$state)
         $state['Attributes'] = $attributes;
 
         \SimpleSAML\Auth\Source::completeAuth($state);
+
+        return $attributes;
```

Comment:
If this were to match the phpdoc, it should be:
return array($username, $attributes);